lib/hanami/helpers/html_helper/empty_html_node.rb

```diff
@@ -124,7 +124,11 @@ def boolean_attribute(attribute_name, _value)
         end
 
         def attribute(attribute_name, value)
-          %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
+          if attribute_name.to_sym == :value
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{Hanami::Utils::Escape.html(value)}")
+          else
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
```

Comment:
Alternatively we can make this change in `_value`: https://github.com/hanami/helpers/blob/8f10dbd7475bd28f6197bd667540577cccad1504/lib/hanami/helpers/form_helper/form_builder.rb#L984

WDYT?